skytemple_randomizer/randomizer/blind_items_moves.py

```diff
@@ -255,17 +256,32 @@
 class BlindItemsMovesRandomizer(AbstractRandomizer):
     def step_count(self) -> int:
         steps = 0
-        if self.config["item"]["blind_items"]["enable"]:
-            steps += 1
-        if self.config["pokemon"]["blind_moves"]["enable"]:
+        if (
+            self.config["item"]["blind_items"]["enable"]
+            or self.config["pokemon"]["blind_moves"]["enable"]
+        ):
             steps += 1
+            if self.config["item"]["blind_items"]["enable"]:
+                steps += 1
+            if self.config["pokemon"]["blind_moves"]["enable"]:
+                steps += 1
         return steps
 
     def run(self, status: Status):
-        if self.config["item"]["blind_items"]["enable"]:
-            self.blind_items(status)
-        if self.config["pokemon"]["blind_moves"]["enable"]:
-            self.blind_moves(status)
+        patcher = Patcher(self.rom, self.static_data)
+
+        if (
+            self.config["item"]["blind_items"]["enable"]
+            or self.config["pokemon"]["blind_moves"]["enable"]
+        ):
```

Comment:
Technically, DisableTips is only needed when blind items are enabled. There's no tips for moves.
Probably doesn't matter though.